### DISCUSSION TITLE
194

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#167] [General]
# 潮流周刊第194期 - 暮色落霞

封面图拍摄于前几天随手拍，最近杭州的夕阳都非常好看，恰逢刚好也没有这么热，还有些风，散散步挺舒服的。

https://weekly.tw93.fun/posts/194-%E6%9A%AE%E8%89%B2%E8%90%BD%E9%9C%9E

**Comment by Liushitong** [comment]
微信读书本身有个缺点：阅读时主体宽度没有办法自适应到全屏。导致微信读书网页版只适合全屏使用，半屏使用左右两边有很多不必要保留的空间。这个问题在使用Pake打包的时候有没有好的办法解决呢？

**Reply by tw93** [reply]
其实可以去试试修改样式
